api/src/main/java/org/pmiops/workbench/db/dao/WorkspaceDao.java

```diff
@@ -24,6 +25,9 @@ Workspace findByWorkspaceNamespaceAndFirecloudNameAndActiveStatus(
   Workspace findByWorkspaceNamespaceAndNameAndActiveStatus(
       String workspaceNamespace, String name, short activeStatus);
 
+  @Query("SELECT distinct w.workspaceNamespace, w from Workspace w")
+  Set<String> findAllWorkspaceNamespaces();
+
   @Query(
```

Comment:
this looks like it's using Java-ified column names like `workspaceNamespace` and `firecloudName` instead of the versions with the underscores.

`select workspace_namespace, firecloud_name, active_status from workspace LIMIT 3;` works for me. Were the columns renamed at some point?

Reply:
Unless it says `native=true` in the query annotation, these are JPQL, not MySQL statements. https://docs.spring.io/spring-data/jpa/docs/current/reference/html/

Reply:
Thank you, I thought I was losing my mind there for a minute.